app/models/user.js

```diff
@@ -1,11 +1,12 @@
-import Model, { attr, belongsTo } from '@ember-data/model';
+import Model, { attr, belongsTo, hasMany } from '@ember-data/model';
 
 export default class UserModel extends Model {
   @attr() firstName;
   @attr() familyName;
 
   @belongsTo('account') account;
   @belongsTo('user-group') group;
+  @hasMany('press-release-activity', { inverse: 'creator' }) pressReleaseActivities;
```

Comment:
Aangezien er maar 1 relatie is vanuit `user` naar `press-release-activity` moet je de inverse niet specifiëren